script/deploy/_targetState.json

```
@@ -959,6 +958,35 @@
       }
     }
   },
+  "fraxtal": {
```

Biome (1.9.4)

[error] The key fraxtal was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)